src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.container {
+  width: fit-content;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  padding-right: 4px;
```

Comment:
Don't use paddings to align items inside flex-box.

In your particular case `padding-right: 4px` has no effect as you set the `width` explicitly.
So as a result you get non-optimal item sizing (notice - items in figma are 16x16, not 16x20)

The solution here is:
1. set item witdth to 16px
2. use margins to align items as you need inside flex container

Reply:
Fixed